frontend/src/components/review/ExistingComment.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState, useContext } from "react";
+import { useMutation } from "@apollo/client";
+import { Text, Flex, Button } from "@chakra-ui/react";
+import AuthContext from "../../contexts/AuthContext";
+import WIPComment from "./WIPComment";
+import {
+  UPDATE_COMMENT_BY_ID,
+  UpdateCommentResponse,
+} from "../../APIClients/mutations/CommentMutations";
+
+export type ExistingCommentProps = {
+  id: number;
+  resolved: boolean;
+  content: string;
+  time: string;
+  commentStoryTranslationContentId: number;
+  lineIndex: number;
+};
+
+const ExistingComment = ({
+  id,
+  resolved,
+  content,
+  time,
+  commentStoryTranslationContentId,
+  lineIndex,
+}: ExistingCommentProps) => {
+  const handleError = (errorMessage: string) => {
+    // eslint-disable-next-line no-alert
+    alert(errorMessage);
+  };
+
+  const [reply, setReply] = useState(0);
```

Comment:
I'm trying to wrap my head around `reply` and `commentLine`: are you treating `reply` as a boolean/flag for displaying a `WIPComment`?